.changeset/pink-jokes-talk.md

```diff
@@ -0,0 +1,11 @@
+---
+'braid-design-system': patch
+---
+
+Drop lodash usage to decrease bundle size
+
+Affected components:
+
+- All field components
+- All Menu components
```

Comment:
For ease of scanning, i'd call out names here, `OverflowMenu` and `MenuRenderer`, and also maybe call out `MonthPicker` as a separate line item.